blocks/library/freeform/editor.scss

```diff
@@ -104,6 +104,16 @@
 		margin-left: auto;
 		margin-right: auto;
 	}
+
+	.wp-more-tag {
+		width: 96%;
+		height: 0;
+		display: block;
+		margin: 15px auto;
+		outline: 0;
+		cursor: default;
+		border: 2px dashed rgb( 186, 186, 186 );
```

Comment:
Where did this color come from? Should it be expressed as a hex value? Should it be one of the standard grays in our variables set?

Reply:
It came from the original image, just to get a sense of how switching away from the image would look, but yeah, I'd rather merge this with a proper var from our set.